packages/yarnpkg-core/sources/nodeUtils.ts

```diff
@@ -4,6 +4,8 @@ import os             from 'os';
 import * as execUtils from './execUtils';
 import * as miscUtils from './miscUtils';
 
+export const major = Number(process.versions.node.split(`.`)[0]);
```

Comment:
I don't think adding this here is necessary when it's only used in tests.

Reply:
I still prefer it to duplicating the code, and nodeUtils is as good a place as any. I'd prefer to avoid adding test-specific "general utils", as they are harder to discover.